### PR TITLE
add canvas trigger function for tools

### PR DESCRIPTION
**Pull request by ruirui19**
Add canvas trigger event to monitor when drawing on mouse up

Ticket: 
https://kiddom.atlassian.net/browse/CE-231?atlOrigin=eyJpIjoiMGY0ZDc5NTk5MzVjNDA0ZGJjOTk2YTM2NDI1YWExM2IiLCJwIjoiaiJ9

### CODE REVIEW

File: src/circle.js [hunk 1]
```diff
@@ -47,7 +47,9 @@ class Circle extends FabricCanvasTool {
   }
 
   doMouseUp(o) {
+    let canvas = this._canvas
     this.isDown = false;
+    canvas.trigger("object:modified", { target: this.line });
```
Comment (nguyen-dunn) [inline src/circle.js:52]: @ruirui19 
this.circle not line correct?

File: src/rectangle.js [hunk 2]
```diff
@@ -58,7 +58,9 @@ class Rectangle extends FabricCanvasTool {
   }
 
   doMouseUp(o) {
+    let canvas = this._canvas
     this.isDown = false;
+    canvas.trigger("object:modified", { target: this.line });
```
Comment (nguyen-dunn) [inline src/rectangle.js:63]: @ruirui19 
this.rect not line correct?

File: package-lock.json [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@kiddom/react-sketch",
-  "version": "0.11.1",
+  "version": "0.11.2",
```
Comment (nguyen-dunn) [inline package-lock.json:3]: I think this version is wrong it should be at least 4 or 5

**Review by nguyen-dunn — CHANGES_REQUESTED** [review]
@ruirui19 I think the version is wrong for this version and other comments

File: package-lock.json (hunk 3 above)
Comment (ruirui19) [reply package-lock.json:3]: I pull the latest master and the current version is 0.11.2..but somehow the package-lock file is not updated

**Review by nguyen-dunn — APPROVED** [review]
👍

**Pull request merged**